packages/back-end/src/models/ApiKeyModel.ts

```diff
@@ -18,3 +18,11 @@ export const ApiKeyModel = mongoose.model<ApiKeyDocument>(
   "ApiKey",
   apiKeySchema
 );
+
+export async function findOrgIdByAccessToken(accessToken: string) {
+  const doc = await ApiKeyModel.findOne({
+    key: accessToken,
+    environment: "access",
+  });
+  return doc?.organization;
+}
```

Comment:
I don't like using `environment` to differentiate between public and secret keys that are used for entirely separate purposes.  I think an explicit property would be better that says what type of key it is.